realsense2_description/urdf/_d435.gazebo.xacro

```diff
@@ -51,7 +51,7 @@ This is the Gazebo URDF model for the Intel RealSense D435 camera
       </sensor>
       <sensor name="${camera_name}ired1" type="camera">
         <camera name="${camera_name}">
-          <horizontal_fov>${85.2*deg_to_rad}</horizontal_fov>
+          <horizontal_fov>${69.4*deg_to_rad}</horizontal_fov>
```

Comment:
This is not correct for a D435. Instead I think we should make a d415 or something.

Reply:
are d435 and d415 in sim identical other than the fov?